firmware/tunerstudio/rusefi.input

```diff
@@ -1287,6 +1287,7 @@ gaugeCategory = Idle
 	idleStatus_outputGauge = idleStatus_output,"Idle PID output", "", -10000.0,10000.0, -10000.0,10000.0, -10000.0,10000.0, 3,3
 	idleStatus_errorGauge = idleStatus_error,"Idle PID error", "", -10000.0,10000.0, -10000.0,10000.0, -10000.0,10000.0, 3,3
 	idleStatus_resetCounterGauge = idleStatus_resetCounter,"Idle PID resetCounter", "", -10000.0,10000.0, -10000.0,10000.0, -10000.0,10000.0, 0,0
+	targetRpmGauge = targetRpm, "Idle Target RPM - engine speed","RPM", 0, {rpmHardLimit + 2000}, 200, {cranking_rpm}, {rpmHardLimit - 500},  {rpmHardLimit},	0,	0
```

Comment:
We define `GAUGE_NAME_IDLE_RPM_TARGET` then don't use it here?

Should also use it at `idle_state.txt` line 31 to prevent duplication of the same string twice